File: Java/백준/Silver/1476. 날짜 계산/날짜 계산.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner s = new Scanner(System.in);

        int E = s.nextInt(); //지구
        int S = s.nextInt(); //태양
        int M = s.nextInt(); //달

        int Earth = 1;
        int Sun = 1;
        int Moon = 1;

        int year = 1;

        while(true){
            if(Earth > 15){ Earth = 1; }
            if(Sun > 28){ Sun = 1; }
            if(Moon > 19){ Moon = 1; }
            if ((Earth == E) && (Sun == S) && (Moon == M)){
                break;
            }
            Earth++;
            Sun++;
            Moon++;
            year++;
        }

        System.out.print(year);
    }
}

File: C11/백준/Silver/1476. 날짜 계산/날짜 계산.c
#include <stdio.h>

int main(void){
	
	int E = 0;
	int S = 0;
	int M = 0;
	
	scanf("%d %d %d", &E, &S, &M);
	
	int Earth = 1;
	int Sun = 1;
	int Moon = 1;
	
	int year = 1;
	
	while(1){
		if(Earth > 15){ Earth = 1; }
		if(Sun > 28){ Sun = 1; }
		if(Moon > 19){ Moon = 1; }
		if((Earth == E) && (Sun == S) && (Moon == M)){
			break;
		}
		Earth++;
		Sun++;
		Moon++;
		year++;
	}
	
	printf("%d", year);
	
	return 0;
}